src/components/DeeplinkWrapper/index.website.js

```diff
@@ -0,0 +1,155 @@
+import _ from 'underscore';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import deeplinkRoutes from './deeplinkRoutes';
+import FullScreenLoadingIndicator from '../FullscreenLoadingIndicator';
+import TextLink from '../TextLink';
+import * as Illustrations from '../Icon/Illustrations';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import CONST from '../../CONST';
+import CONFIG from '../../CONFIG';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import colors from '../../styles/colors';
+import * as Browser from '../../libs/Browser';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.node.isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+class DeeplinkWrapper extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            appInstallationCheckStatus: this.isMacOSWeb() ? 'checking' : 'not-installed',
+        };
+    }
+
+    componentDidMount() {
+        if (!this.isMacOSWeb()) {
+            return;
+        }
+
+        let focused = true;
+
+        window.addEventListener('blur', () => {
+            focused = false;
+        });
+
+        setTimeout(() => {
+            if (!focused) {
+                this.setState({appInstallationCheckStatus: 'installed'});
+            } else {
+                this.setState({appInstallationCheckStatus: 'not-installed'});
+            }
+        }, 500);
+
+        // check if pathname matches with deeplink routes
+        const pathname = window.location.pathname;
```

Comment:
Arguably doesn't need to be its own variable, could just be inline on line 58 

```Javascript 
            return routeRegex.test(window.location.pathname);
```

Reply:
Done